braid/braid.c

```diff
@@ -65,6 +65,8 @@ braid_Drive(braid_Core  core)
       _braid_AdjointFeatureCheck(core);
    }
 
+   /* TODO: check for unsupported Delta features */
```

Comment:
Do we want to add checks here, like init_basis must exist, if delta==True?

Reply:
I think that's a great idea. This is why I left a TODO in there. I was sure you guys would have better ideas for things to check for.